addons/html_builder/static/src/core/building_blocks/builder_urlpicker.js

```diff
@@ -0,0 +1,73 @@
+import { BuilderComponent } from "@html_builder/core/building_blocks/builder_component";
+import {
+    BuilderTextInputBase,
+    textInputBasePassthroughProps,
+} from "@html_builder/core/building_blocks/builder_text_input_base";
+import {
+    basicContainerBuilderComponentProps,
+    useBuilderComponent,
+    useInputBuilderComponent,
+} from "@html_builder/core/building_blocks/utils";
+import { Component, useEffect } from "@odoo/owl";
+import { useChildRef } from "@web/core/utils/hooks";
+import { pick } from "@web/core/utils/objects";
+import wUtils from "@website/js/utils";
```

Comment:
I'm wondering if we should find a way to decouple the dependencies of html_builder to website. Especially in builder components.